jekyll/_cci2/pipelines.adoc

```diff
@@ -0,0 +1,138 @@
+** Authorize your connection if this is not already showing with a icon:check-circle[]
+** Select your project repository
+** Select a fallback branch to use in case you try to build on a branch with no configuration defined
+** Enter the path to your configuration file relative to the root of your repository. For example, `.circleci/config.yml`
+. Click btn:[Save]
+
+Once you have set up a configuration source you need to set up a project trigger to connect it to. This is described in the following section.
+
+=== Project triggers
+
+TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a trigger is already set up.
+
+A project can have one or more triggers, which are associated with events from a source of change. Triggers include, but are not limited to, a push to a branch on a version control system (VCS), for example, a GitHub repository. A trigger determines an event (or events) along with a configuration source to define a pipeline.
+
+A trigger kicks off a pipeline to run the workflows defined by your connected configuration source.
+
+image::{{site.baseurl}}/assets/img/docs/project-settings-triggers.png[Triggers]
+
+==== GitHub App project triggers
+
+When the CircleCI GitHub App is installed for your organization, GitHub starts to send events for the repositories you have granted access to. When a trigger is created CircleCI has enough information to use the event data to determine if a pipeline should be triggered.
+
+==== GitLab project triggers
+
+When a trigger is created, CircleCI registers a webhook with GitLab. Push events from GitLab are sent to CircleCI. CircleCI then uses the event data to determine _if_ a pipeline should run, and if so, _which_ pipeline should be run.
+
+In addition to a configuration source, each trigger includes the webhook URL, and in this scenario, a CircleCI-created GitLab token. The webhook URL and GitLab token are used to securely register the webhook within GitLab in order to receive push events from your GitLab repository.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-edit-trigger.png[Trigger details]
+
+**Trigger filters** allow you to determine when a trigger should initiate a build based on the parameters provided by Gitlab’s webhook. CircleCI provides some common options, for example, only build on merge requests, but you can also build your own rules using the custom filter option. For example, a custom filter would allow you to only build on a specific branch or user.
+
+image::{{site.baseurl}}/assets/img/docs/gl-preview/gitlab-preview-project-settings-customize-triggers.png[Trigger details]
+
```

Comment:
We don't have a subsection "Inbound Webhook triggers", is it intentional?

Reply:
Good point!

Reply:
I'll see if I have the info to make one